src/Symfony/Component/Cache/Tests/Adapter/ApcuAdapterTest.php

```diff
@@ -15,6 +15,9 @@
 use Psr\Log\NullLogger;
 use Symfony\Component\Cache\Adapter\ApcuAdapter;
 
+/**
+ * @group time-sensitive
```

Comment:
will this have any effect? apcu does its own internal time comparisons which we cannot mock, so I'm not sure this works. WDYT?

Reply:
reverted